### ISSUE TITLE
Unable to start RsyncServer service

### ISSUE THREAD

**Issue by ossama-othman** [backuppc/cygwin-rsyncd#8]
### Software
**Release**: [cygwin-rsyncd-3.2.3.0](https://github.com/backuppc/cygwin-rsyncd/releases/tag/3.2.3.0)
**OS**:  Microsoft Windows 10 Pro (10.0.19041 Build 19041)

### Problem Description
After running the cygwin-rsyncd installer, the `RsyncServer` service fails to start.  Attempting to manually start the service also fail.  It looks like A cygwin DLL is missing. From `C:\rsyncd\rsyncd-stderr`:

```
C:/rsyncd/bin/rsync.exe: error while loading shared libraries: cygxxhash-0.dll: cannot open shared object file: No such file or directory
```

**Comment by craigbarratt** [comment]
Thanks for the report.  I'd like to make sure there aren't any other missing libraries.  Can you install `cygxxhash-0.dll` and re-run to make sure there aren't any other errors?

If it's not easy for you to find `cygxxhash-0.dll`, I can push a copy to GitHub so you can download it from there.

**Comment by craigbarratt** [comment]
I added the five missing libraries.  I've attached a candidate new installer.  It's a zip file that contains the exe (I can't upload an exe file).  Could you try that out please?

[cygwin-rsyncd-3.2.3.1_installer.zip](https://github.com/backuppc/cygwin-rsyncd/files/5955605/cygwin-rsyncd-3.2.3.1_installer.zip)

**Comment by ossama-othman** [comment]
I tried the new 3.2.3.1 installer in the ZIP file .  The `RsyncServer` service now successfully starts.

Thanks for getting back with a fix so quickly!

**Comment by craigbarratt** [comment]
Thanks for confirming.  I just released 3.2.3.1.
